data/remnant/remnant events.txt

```diff
@@ -287,7 +287,7 @@ event "remnant: ssil vida inhabited"
 		landscape land/station0
 		description `The silence of this station rings in your ears, and as you explore you are increasingly surprised that it even has an atmosphere. It is obviously ancient, and patches of ambiguous green and purple material resembling fungus grow through the walls. Whoever built this station designed it for beings much larger than you - either that, or they liked spacious environments. Even the shortest doorway you pass through is easily double your height. They are always wider than they are tall.`
 		description `	Exploring beyond the docks reveals that most of this facility is infrastructure for some type of massive emitter. It is not clear exactly what it's supposed to be emitting, but it might have something to do with how this system appeared in the first place.`
-		spaceport `The Remnant have set up a fully self-contained base of operations in this part of the station. Independent life support, its own power system, and controls tapping into the rest of station occupy much of the space, while a research laboratory and additional equipment fill up the rest. A rudimentary cafeteria, communication links, basic repair equipment, and sleeping quarters round out the amenities of this research camp.`
+		spaceport `The Remnant have set up a fully self-contained base of operations in this part of the station. Independent life support, its own power system, and controls tapping into the rest of station occupy much of the space, while a Research Laboratory and additional equipment fill up the rest. A rudimentary cafeteria, communication links, basic repair equipment, and sleeping quarters round out the amenities of this research camp.`
```

Comment:
```suggestion
		spaceport `The Remnant have set up a fully self-contained base of operations in this part of the station. Independent life support, its own power system, and controls tapping into the rest of station occupy much of the space, while a research laboratory and additional equipment fill up the rest. A rudimentary cafeteria, communication links, basic repair equipment, and sleeping quarters round out the amenities of this research camp.`
```
I'm unsure about this one, but I'll point it out anyway.

Reply:
I went back and forth on this one myself, but as it seems they kind of unpacked a ships internals to set up in the station, I think it's best either as is or with a wording tweak.

Reply:
General comment: it might be a good idea to introduce a rule that says outfits are ALWAYS capitalized in descriptions/names (without it, it kinda blends in and one can't tell it's an actual outfit's name)

Or have it be displayed in bold/italic/some other font, but I'm very much not sure if ES can do that...

Reply:
> I went back and forth on this one myself, but as it seems they kind of unpacked a ships internals to set up in the station, I think it's best either as is or with a wording tweak.

Well, I hadn't considered the ship angle, so I reckon this is alright then, no change needed.